src/sid/covid_epi_params.csv

```diff
@@ -63,6 +58,7 @@ cd_symptoms_false,80-100,18,0.2139
 cd_symptoms_false,80-100,22,0.2139
 cd_symptoms_false,80-100,27,0.2139
 cd_symptoms_false,80-100,-1,0.287
+
```

Comment:
```suggestion
```

Reply:
Couldn't accept your suggestion here but removed the blank line.